frontend/src/pages/answer/kgBubble/KgBubble.jsx

```diff
@@ -0,0 +1,111 @@
+/* eslint-disable indent, no-use-before-define, func-names, no-return-assign */
```

Comment:
I don't think this would work out, because a node bubble can be referenced in multiple results. But I think it could filter the results and only show the ones that contain that node. Created #188